conda_forge_automerge_action/automerge.py

```diff
@@ -445,7 +445,7 @@ def _automerge_pr(repo, pr):
     try:
         merge_status = pr.merge(
             commit_message="automerged PR by conda-forge/automerge-action",
-            commit_title=pr.title,
+            commit_title=f"{pr.title} ({pr.id})",
```

Comment:
Does this need a # in front of the id?

Reply:
Yeah, you're right, I messed that one up when shifting it from the commit message to the title. 🙈